### PR TITLE
feat: smaller onboarding banner

### PR DESCRIPTION
**Pull request by omBratteng**
AS-390

<table>
 <tr>
  <td>Before
  <td>After
 <tr>
  <td>
  <td>
</table>

### Preview domain
https://as-390.preview.app.daily.dev

### CODE REVIEW

**Review by nensidosari — APPROVED** [review]
looks good

**Review by rebelchris — APPROVED** [review]
Looks good to me.

File: packages/shared/src/lib/featureManagement.ts [hunk 1]
```diff
@@ -43,6 +44,7 @@ const feature = {
   searchSources: new Feature('search_sources', false),
   onboardingChecklist: new Feature('onboarding_checklist', false),
   feedLayoutPreview: new Feature('feed_layout_preview', false),
+  onboardingBanner: new Feature('onboarding_banner', OnboardingBanner.Control),
```
Comment (omBratteng) [inline packages/shared/src/lib/featureManagement.ts:47]: @rebelchris could you create this in GB?

Comment (rebelchris) [reply packages/shared/src/lib/featureManagement.ts:47]: done

**Pull request merged**